projectional/src/main/java/jetbrains/jetpad/projectional/generic/CollectionEditor.java

```diff
@@ -28,12 +29,15 @@
   private List<ViewT> myViews;
   private Property<ItemT> myForDeletion;
   private boolean myCanCreateNew;
+  private Predicate<ViewT> myReplaceWithNewOnDel;
 
-  protected CollectionEditor(List<ItemT> items, List<ViewT> views, Property<ItemT> forDeletion, boolean canCreateNew) {
+  protected CollectionEditor(List<ItemT> items, List<ViewT> views, Property<ItemT> forDeletion, boolean canCreateNew,
+      Predicate<ViewT> replaceWithNewOnDel) {
```

Comment:
We don't need this parameter. We pass canCreateNew. We can inline this predicate. 
Then we wouldn't have to name it (otherwise it should be replaceWithNewOnDelete - we usually don't shorten words).